gpu/api_test.go

```diff
@@ -304,13 +314,14 @@ func Test_ScryptPositions_Pow(t *testing.T) {
 				WithComputeProviderID(p.ID),
 				WithCommitment(commitment),
 				WithSalt(salt),
-				WithStartAndEndPosition(0, 128*1024),
-				WithBitsPerLabel(8),
+				WithStartAndEndPosition(0, 256*1024),
+				WithBitsPerLabel(8*16),
```

Comment:
Would it make sense to change this parameter to `WithBytesPerLabel` and stop supporting label sizes that are not full bytes?

Reply:
Perhaps 🤷 The gpu-post supports it so it seems to me unnecessary to artificially limit it.